### PR TITLE
adding Redis URL

### PR DESCRIPTION
**Pull request by wejdross**
## Summary

* Adding comp-function to generate REDIS_URL as described in docs: https://docs.appcat.ch/references/secrets.html#_redis

### CODE REVIEW

File: pkg/comp-functions/functions/vshnredis/redis_url.go [hunk 1]
```diff
@@ -0,0 +1,80 @@
...
+	log := controllerruntime.LoggerFrom(ctx)
+
+	comp := &vshnv1.VSHNRedis{}
+	err := iof.Desired.GetComposite(ctx, comp)
+	if err != nil {
+		return runtime.NewFatalErr(ctx, "Cannot get composite from function io", err)
+	}
+
+	// Wait for the next reconciliation in case instance namespace is missing
+	if comp.Status.InstanceNamespace == "" {
```
Comment (Kidswiss) [inline pkg/comp-functions/functions/vshnredis/redis_url.go:43]: Not sure why you're waiting for this status here?

You don't seem to be reading the value for the secret itself?

Comment (Kidswiss) [inline pkg/comp-functions/functions/vshnredis/redis_url.go:43]: Not sure why you're waiting for this status here?

You don't seem to be reading the value for the secret itself?

**Review by Kidswiss — APPROVED** [review]
LGTM

Just one nitpick that confuses me a bit.

**Review by zugao — APPROVED** [review]
LGTM

File: pkg/comp-functions/functions/vshnredis/redis_url.go (hunk 1 above)
Comment (wejdross) [reply pkg/comp-functions/functions/vshnredis/redis_url.go]: deleted it, leftover from copy-paste ;p

**Pull request merged**